octobot_trading/data/trade.py

```diff
@@ -65,3 +69,19 @@ def update_from_order(self, order, creation_time=0, canceled_time=0, executed_ti
         self.canceled_time = order.canceled_time if order.canceled_time > 0 else canceled_time
         self.executed_time = order.executed_time if order.executed_time > 0 else executed_time
         self.symbol = order.symbol
+
+    def to_dict(self):
+        trade_time = self.executed_time if self.status is not OrderStatus.CANCELED else self.canceled_time
+        return {
+            ExchangeConstantsOrderColumns.ID.value: self.trade_id,
+            ExchangeConstantsOrderColumns.SYMBOL.value: self.symbol,
+            ExchangeConstantsOrderColumns.PRICE.value: self.executed_price,
+            ExchangeConstantsOrderColumns.STATUS.value: self.status.value,
+            ExchangeConstantsOrderColumns.TIMESTAMP.value: trade_time,
+            ExchangeConstantsOrderColumns.TYPE.value: self.exchange_trade_type.value,
```

Comment:
Why it's `order.exchange_order_type` and not `order.order_type` ?

Reply:
For clarity, considering the above comment, maybe we should rename ```TradeOrderType``` as ```ExchangeTradeOrderType```